PopulateTools/gobierto
# Reset query and viz

Closes #2916 #2970


## :v: What does this PR do?

Include button for reset query from editor.
Include button for reset Visualization.

## :mag: How should this be manually tested?
Staging

Steps to reset query:

1. Edit the query on editor.
2. Click on the reset query button.
3. The default query will appear in the editor

Steps to reset saved query:

1. Load a saved query.
2. Edit the saved query on editor.
3. Click on the reset query button.
4. The saved query will appear in the editor.

Steps to reset Visualization:

1. Click on Visualize/Visualizar
2. Select a new type of Visuzaliton.
3. Saved Visualization and reset visualization buttons will appear, the Visualize button will disappear.
4. Click on the reset visualization button.
5. Perspective visualization changes to Grid, and Perspective menus shouldn't be seen.
